src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt

```diff
@@ -14,10 +15,12 @@ import org.springframework.web.reactive.function.server.ServerResponse
 import org.springframework.web.reactive.function.server.ServerResponse.*
 import org.springframework.web.reactive.function.server.bodyToMono
 import reactor.core.publisher.Mono
+import javax.mail.SendFailedException
 
 @Component
 class DataAccessRequestHandler(
-    private val dataAccessRequestRepository: DataAccessRequestRepository
+    private val dataAccessRequestRepository: DataAccessRequestRepository,
+    private val emailService: EmailService
```

Comment:
I think is beter to inject to use this sevruce in a data access request service when there's going to be one

Reply:
I don't understand what you meant here

Reply:
Sorry for the typo, I meant that we need to create a service for the data access request